pages/mesosphere/dcos/services/dse/3.1.0-6.7.5/release-notes/index.md

```diff
@@ -1,13 +1,20 @@
 ---
 layout: layout.pug
 navigationTitle: Release Notes
-excerpt: Release Notes for DSE v. 3.0.0-6.7.2
+excerpt: Release Notes for DSE v. 3.1.0-6.7.5
 title: Release Notes
 menuWeight: 10
 model: /mesosphere/dcos/services/dse/data.yml
 render: mustache
 ---
 
+# Version 3.1.0-6.7.5
+
+## Updates
+- Upgraded DSE to version `6.7.5` and OpsCenter to version `6.7.4`
+- Upgraded SDK to `0.57.1`. For more information see [SDK Release Notes](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.1)
```

Comment:
There's been a few releases since the last `55.4`, especially with `57.0` being a significant update. Can we include a list of all the releases since then? Something like:
```
- Upgraded SDK to `0.57.1`. For more information see SDK's Release Notes for: 
  - [0.57.1](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.1)
  - [0.57.0](...)
  - [0.56.x](...)

Reply:
updated as suggested.